Source/ACE.Database/ShardDatabase.cs

```diff
@@ -387,6 +387,15 @@ public List<Biota> GetWieldedItemsInParallel(uint parentId)
             return wieldedItems.ToList();
         }
 
+        public Biota GetObjectByGuid(uint guid)
```

Comment:
Why the addition of GetObjectByGuid. This seems to do the same thing as GetBiota, except it returns a biota that doesn't use our context caching system.